src/databricks/labs/ucx/hive_metastore/table_migration_status.py

```diff
@@ -153,10 +155,19 @@ def _try_fetch(self) -> Iterable[TableMigrationStatus]:
         for row in self._fetch(f"SELECT * FROM {escape_sql_identifier(self.full_name)}"):
             yield TableMigrationStatus(*row)
 
+    def _iter_catalogs(self) -> Iterable[CatalogInfo]:
+        try:
+            yield from self._ws.catalogs.list()
+        except DatabricksError as e:
+            logger.warning("Cannot list catalogs", exc_info=e)
+
```

Comment:
Is it necessary? Is it a warning or a failure? How likely are we to not be able to list catalogs and still succeed?